preinstall/main.go

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright 2018 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"flag"
+)
+
+var (
+	namespace     string
+	csrName       string
+	tlsSecretName string
+)
+
+func init() {
+	flag.StringVar(&namespace, "namespace", "default", "The namespace you want to install kritis in.")
```

Comment:
You can get this by reading /var/run/secrets/kubernetes.io/serviceaccount/namespace instead.